sql/files/defaultdata/java_javac_detect/run

```diff
@@ -56,9 +56,9 @@ rm -f $TMPFILE
 
 if [ -z "$MAINCLASS" ]; then
 	# Look for class that has the 'main' function:
-	CLASSNAMES="$(find ./* -type f -regex '^.*\.class$' \
+	CLASSNAMES="$(find ./* -type f -regex '^.*\.class$' ! -name DetectMain.class \
```

Comment:
this looks like something we should do regardless of this change, right?

Reply:
No it’s only now a problem because we copy this class in, before we included it from a different part of the fs but of course this is no longer readable when in the chroot